phylanx/performance/phyprof/phyprof/computation.py

```diff
@@ -0,0 +1,32 @@
+__license__ = """ 
```

Comment:
I'm still a little unclear on the status of phyprof. Is it a working tool, or is it more of a prototype/idea?

Reply:
It's a placeholder for [Phylanx](https://github.com/STEllAR-GROUP/phylanx/) performance module built on top of [APEX](https://github.com/khuck/xpress-apex) and [Traveler](https://github.com/hdc-arizona/traveler-integrated).